build.gradle

```diff
@@ -325,7 +325,7 @@ startScripts {
 }
 
 dependencies {
-  compile project(':ethsigner')
+  compile project(':ethsigner:app')
```

Comment:
Not convinced there is much value is a seperate module for app. Do we need it?

Reply:
Not really sure - figured while I was messing around in here then I could do this ... its not a BAD thing to split out ... just maybe overkill